pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-name = "lephare"
+name = "lephare-dev"
```

Comment:
Temporary change required by pypi. It requires that the name be the same as the repository name. We'll change this back when the repo name changes to lephare. I created this issue so we don't forget: https://github.com/lincc-frameworks/lephare-dev/issues/43